app/i18n/locales/ar.js

```diff
@@ -101,7 +101,7 @@ export default {
 	Announcement: 'إعلان',
 	Apply_Your_Certificate: 'طبق شهادتك',
 	Applying_a_theme_will_change_how_the_app_looks: 'سيؤدي تطبيق السمة إلى تغيير شكل التطبيق',
-	ARCHIVE: 'أرشفة',
+	ARCHIVE: 'أرشيف',
```

Comment:
This is a call to action to `Archive` a channel.
It's not a list of things.

Reply:
ahhh i tought its list